subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmComponentDependencies.java

```diff
@@ -116,4 +116,28 @@ public interface JvmComponentDependencies {
      * @since 7.5
      */
     void annotationProcessor(Object dependencyNotation, Action<? super Dependency> configuration);
+
+    /**
+     * Creates a dependency on the API of the current version of Gradle.
+     *
+     * @return The dependency.
+     * @since 7.6
+     */
+    Dependency gradleApi();
+
+    /**
+     * Creates a dependency on the <a href="https://docs.gradle.org/current/userguide/test_kit.html" target="_top">Gradle test-kit</a> API.
+     *
+     * @return The dependency.
+     * @since 7.6
+     */
+    Dependency gradleTestKit();
+
+    /**
+     * Creates a dependency on the Groovy that is distributed with the current version of Gradle.
```

Comment:
`Groovy version`? `Groovy implementation`?

Reply:
Have to use the GAV for that (and should probably prefer it in most cases anyway).  This method doesn't take a version even at the top-level - the version you get depends on the version of Gradle you're running.

Reply:
I think this was a question about the phrasing in the Javadoc, you say "the Groovy"  and it sounds like there should be another word there.

Reply:
Yeah Octavia is right. I should have used more words here as well